CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required (VERSION 3.2)
+project (benchmark C)
+
+# Default cross compile settings
+set (CMAKE_TOOLCHAIN_FILE CMakeToolchain.txt)
```

Comment:
should `CMakeToolchain.txt` be also created?

Reply:
Sorry, missed that.